vision_datasets/image_object_detection/detection_as_kvp_dataset.py

```diff
@@ -0,0 +1,113 @@
+import logging
+import typing
+from copy import deepcopy
+
+from vision_datasets.common import DatasetTypes, KeyValuePairDatasetInfo, VisionDataset
+from vision_datasets.key_value_pair import (
+    KeyValuePairDatasetManifest,
+    KeyValuePairLabelManifest,
+)
+
+logger = logging.getLogger(__name__)
+
+
+BBOXES_KEY = "bboxes"
+BASE_DETECTION_SCHEMA = {
+    "name": "Image Object Detection",
+    "description": "Detect objects in images and provide bounding boxes and class label for each object.",
+    "fieldSchema": {
+        f"{BBOXES_KEY}": {
+            "type": "array",
+            "description": "Bounding boxes of objects in the image",
```

Comment:
Maybe something like "Objects in the image of the specified classes, with bounding boxes"?

Reply:
Updated